src/components/textarea/index.tsx

```diff
@@ -18,6 +18,7 @@ interface Props {
   onChange?: ChangeEventHandler<HTMLTextAreaElement>;
   rows?: number;
   cols?: number;
+  size?: 'sm' | 'md';
```

Comment:
Is it not the fontSize that you are changing?
I'm confused when I see size but what it changes is the font and no the texArea size.